var/spack/repos/builtin/packages/mstk/package.py

```diff
@@ -24,33 +24,57 @@ class Mstk(CMakePackage):
     homepage = "https://github.com/MeshToolkit/MSTK"
     git      = "https://github.com/MeshToolkit/MSTK"
 
-    maintainers = ['julienloiseau']
+    maintainers = ['raovgarimella','julienloiseau']
 
+    def url_for_version(self, version):
+        url = 'https://github.com/MeshToolkit/MSTK/archive/{0}.tar.gz'
+        return url.format(version)
```

Comment:
I would drop this function and remove the `v` from the version numbers below, that should work.